### ISSUE TITLE
Sending using data channel on windows fails (incorrect data channel id)

### ISSUE THREAD

**Issue by simonbengtsson** [flutter-webrtc/flutter-webrtc#963]
It is happening since 0.8.5. In 0.8.7 a crash was fixed but the data channel id is still wrong which means that sending fails.

**Workaround**
Commenting out [the line](https://github.com/flutter-webrtc/flutter-webrtc/blob/master/lib/src/native/rtc_data_channel_impl.dart#L54) that assigns the incorrect data channel id.

A fix is available [here](https://github.com/simonbengtsson/flutter-webrtc) and can be used like this

```yaml
flutter_webrtc:
  git:
    url: https://github.com/simonbengtsson/flutter-webrtc
    ref: master
```

* **Flutter version**: 3.0.1
* **Plugin version**: 0.8.7
* **OS**: Windows
* **OS version**: 11

**Comment by davidliu** [comment]
This should be fixed by #961.

**Comment by simonbengtsson** [comment]
Awesome! Can confirm it is working now 🎉
